src/score_hv/harvesters/daily_bfg.py

```diff
@@ -81,10 +169,20 @@ def set_variables(self):
                        "Please reconfigure the input dictionary using only the "
                        "following variables: %r" % (var, VALID_VARIABLES))
                 raise KeyError(msg)
-    
+     
     def get_stats(self):
         ''' return list of all stat types based on harvest_config '''
         return self.stats
+    
+    def get_regions(self):
+        ''' return list of regions based on harvest config'''
+        return self.regions
+
+    def set_regions(self):
+        self.regions = self.config_data.get('region')
```

Comment:
I'm wondering if it would make sense for the input value to be called regions plural since it can be multiple to make that more obvious to the user. Either way, we want to make sure that the readme reflects the expected value here 'region' (as of now) as I could see that being confusing so let's just catch it early.